handler/http/request/book_handler_form_validator.go

```diff
@@ -0,0 +1,19 @@
+package request
+
+import (
+	"github.com/ryota1116/stacked_books/domain/model/googleBooksApi"
+	"github.com/ryota1116/stacked_books/handler/http/response"
+)
+
+type BookHandlerFormValidator struct {
+	GoogleBooksApiRequestBody googleBooksApi.RequestParameter
+}
+
+// Validate : BookHandler.SearchBooksのリクエストボディのバリデーション
+func (bh BookHandlerFormValidator) Validate() (bool, response.ErrorResponseBody) {
+	if bh.GoogleBooksApiRequestBody.Title == "" {
+		return false, response.ErrorResponseBody{Message: "本のタイトルを入力してください"}
```

Comment:
「エラーメッセージをレスポンスクラスに変換する」のはhandlerの責務かなと思いました！

```suggestion
		return false, "本のタイトルを入力してください"
```

Reply:
handler配下のファイルなので問題ないかなと思ったのですが、いかがでしょうか？👀

![image](https://user-images.githubusercontent.com/60560627/136244420-b53a7354-5f70-4865-9401-c489b55c1d95.png)

Reply:
すみません！説明不足でした 🙇 

今の実装だと`request`配下のクラスが`response`に関わる知識を持ってしまっているので、
`handler/book.go`がレスポンスに変換した方がいいかなという意図でした！

Reply:
すみません完全に見落としてました🙇‍♂️
2b9341c でValidationError構造体を作成しました！